mypy/plugins/functools.py

```diff
@@ -229,9 +229,11 @@ def partial_new_callback(ctx: mypy.plugin.FunctionContext) -> Type:
             partial_names.append(fn_type.arg_names[i])
         elif actuals:
```

Comment:
I think this can be simply `else`, since if the above branch is false this is always true.